README.md

```diff
@@ -10,3 +12,71 @@ India currently faces a significant shortage of nurses, with only 2.06 nurses pe
 - Nacto Care facilitates immediate access to home nursing services for individuals in need, particularly the elderly. By leveraging technology, we connect patients with qualified nurses, ensuring timely and personalized care in the comfort of their homes.
 - Similar to how platforms like Zomato and Swiggy have transformed the gig economy, Nacto Care seeks to raise awareness about nursing as a rewarding career choice. Through outreach programs and educational initiatives, we aim to inspire more individuals to pursue nursing as a profession.
 - We prioritize the well-being and satisfaction of our patients by offering additional benefits such as a life insurance scheme. This incentivizes patients to choose Nacto Care for their healthcare needs, fostering trust and loyalty in our services.
+
+# Technologies Used
+
+- The Tech-Stack used in the project are:
+
+  React.js, FastAPI, PostgreSQL, Figma, JavaScript, Python.
+  
+  ![](https://skillicons.dev/icons?i=react,fastapi,postgresql,figma,javascript,python)
+  
+# Installation Steps
+
+1. Clone the repository.
+   ```bash
+   git clone https://github.com/Nactore-Org/Nacto-Care
+   ```
+   You can also [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) the repository.
+
+2. Go to the frontend directory. Install dependencies
```

Comment:
No need to go to frontend directory since there is no such directory. The root of the project is frontend. So change "Go to frontend directory..." to "From the root directory of the project..."

Reply:
> No need to go to frontend directory since there is no such directory. The root of the project is frontend. So change "Go to frontend directory..." to "From the root directory of the project..."
@Harshil-Jani I have made the updations. Can you please assign GSSoC label to the PR please.
#72 
Thank You.